root/hist/hist/RHistFillBenchmarks.cxx

```diff
@@ -0,0 +1,83 @@
+#include "ROOT/RHist.hxx"
+#include "ROOT/RHistConcurrentFill.hxx"
+
+#include "benchmark/benchmark.h"
+
+#include <iostream>
+#include <future>
+
+
+
+// Benchmark for simple Fill on 2D hist
+static void BM_RHist_Fill(benchmark::State &state)
+{
+   ROOT::Experimental::RH2D hist{{100, 0., 1.}, {{0., 1., 2., 3., 10.}}};
+
+   for (auto _ : state) {
+      // Fill without weight
+      for (int i = 0; i < 3000; ++i) {
+         hist.Fill({(double)i / 100, (double)i / 10});
```

Comment:
Maybe simply
```suggestion
         hist.Fill({i / 100., i / 10.});
```
unless I'm missing something? There are more occurrences of this pattern below.